corehq/apps/registration/tasks.py

```diff
@@ -50,3 +51,35 @@ def activation_24hr_reminder_email():
             text_content=message_plaintext,
             email_from=settings.DEFAULT_FROM_EMAIL
         )
+
+
+WIKI_LINK = 'http://help.commcarehq.org'
```

Comment:
Yeah, I definitely don't know the answer to your question. But I'll change it to `https://wiki.commcarehq.org` because I agree it makes more send to link people directly to the wiki